ProjectHeads/App.Head.props

```diff
@@ -40,7 +40,42 @@
       </ItemGroup>
     </When>
   </Choose>
-
+  <Choose>
+    <When Condition="'$(ToolkitSettingsControlsSourceProject)' == ''">
+      <ItemGroup>
+        <PackageReference Include="CommunityToolkit.$(DependencyVariant).Controls.SettingsControls" Version="8.0.230823-rc"/>
+      </ItemGroup>
+    </When>
+    <Otherwise>
```

Comment:
These all need to follow the pattern above, like the Converters:

`<When Condition="'$(IsSingleExperimentHead)' == 'true' and $(MSBuildProjectName.StartsWith('Converters')) == 'false'">`

Reply:
Oh whoops, added that